models/zeroex/base/zeroex_base_api_fills.sql

```diff
@@ -0,0 +1,264 @@
+{{  config(
+    tags=['dunesql'],
+        alias = alias('api_fills_base'),
+        materialized='incremental',
+        partition_by = ['block_date'],
```

Comment:
we're looking to universally increase partitions to `block_month`

if you could introduce a new column next to `block_date` in the script, then you can apply here